examples/js-sub.c

```diff
@@ -130,21 +171,48 @@ int main(int argc, char **argv)
 
     if (s == NATS_OK)
     {
-        if (pull)
+        if (pull && async)
+        {
+            jsOpts.PullSubscribeAsync.MaxMessages = (int) total;
+            jsOpts.PullSubscribeAsync.NoWait = true;
+            jsOpts.PullSubscribeAsync.Timeout = 3600 * 1000; // 1 hour
+            jsOpts.PullSubscribeAsync.FetchSize = 17;
+            jsOpts.PullSubscribeAsync.KeepAhead = 7;
+            jsOpts.PullSubscribeAsync.CompleteHandler = _completeFetchCb;
+            jsOpts.PullSubscribeAsync.CompleteHandlerClosure = NULL;
+
+            // Uncomment to use a 1 second heartbeat.
```

Comment:
Heartbeat should be half of expires.

Reply:
I am not sure what this means, Expires is the lifetime of the sub.